packages/clerk-js/src/core/resources/Session.ts

```diff
@@ -142,7 +144,7 @@ export class Session extends BaseResource implements SessionResource {
   #getCacheId(template?: string, organizationId?: string | null) {
     const resolvedOrganizationId =
       typeof organizationId === 'undefined' ? this.lastActiveOrganizationId : organizationId;
-    return [this.id, template, resolvedOrganizationId, this.updatedAt.getTime()].filter(Boolean).join('-');
+    return [this.id, template, resolvedOrganizationId].filter(Boolean).join('-');
```

Comment:
Key change, updating the cache key to not append updated at since this would cause cache misses in other tabs